tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala

```diff
@@ -129,13 +131,28 @@ object Bindgen {
         withArgs("--extra-arg-before", extraArgBefore) ++
         Seq(header.get.getAbsolutePath, "--")
 
-      var errs = Seq[String]()
-
-      val output = Process(cmd).!!(ProcessLogger { err: String =>
-        errs :+= err
-      })
-
-      new Bindings(output, errs.mkString("\n"))
+      var output = ""
+      var errs   = ""
+      val io = new ProcessIO(
+        _ => {},
+        stdout => {
+          output = Source.fromInputStream(stdout).mkString("")
+          stdout.close()
+        },
+        stderr => {
+          errs = Source.fromInputStream(stderr).mkString("")
+          stderr.close()
+        }
+      )
+      val process   = Process(cmd).run(io)
+      val exitValue = process.exitValue()
+
+      if (exitValue != 0) {
+        System.err.println(
+          s"Bindgen finished with exit code $exitValue.\n$errs")
+      }
+
+      new Bindings(output, errs)
```

Comment:
I've created #144 that I think is a bit cleaner. It also adds messages for the `require` calls which was a problem I had when developing the bindings and sbt plugin.